samples/test/filters.js

```diff
@@ -0,0 +1,777 @@
+            },
+            connected_wifi: {
+              value: 1,
+              timestamp: TIMESTAMP,
+            },
+            os_build: {
+              value: 'PQ2A.190405.004',
+              timestamp: TIMESTAMP,
+            },
+          },
+          cell_plan: {
+            data_plan_05gb: {
+              value: 'true',
+              timestamp: TIMESTAMP,
+            },
+          },
+        },
+      },
+      {
+        key: 'phone#4c410523#20190505',
+        data: {
+          stats_summary: {
+            connected_cell: {
+              value: 0,
+              timestamp: TIMESTAMP,
+            },
+            connected_wifi: {
+              value: 1,
+              timestamp: TIMESTAMP,
+            },
+            os_build: {
+              value: 'PQ2A.190406.000',
+              timestamp: TIMESTAMP,
+            },
+          },
+          cell_plan: {
+            data_plan_05gb: {
+              value: 'true',
+              timestamp: TIMESTAMP,
+            },
+          },
+        },
+      },
+      {
+        key: 'phone#5c10102#20190501',
+        data: {
+          stats_summary: {
+            connected_cell: {
+              value: 1,
+              timestamp: TIMESTAMP,
+            },
+            connected_wifi: {
+              value: 1,
+              timestamp: TIMESTAMP,
+            },
+            os_build: {
+              value: 'PQ2A.190401.002',
+              timestamp: TIMESTAMP,
+            },
+          },
+          cell_plan: {
+            data_plan_10gb: {
+              value: 'true',
+              timestamp: TIMESTAMP,
+            },
+          },
+        },
+      },
+      {
+        key: 'phone#5c10102#20190502',
+        data: {
+          stats_summary: {
+            connected_cell: {
+              value: 1,
+              timestamp: TIMESTAMP,
+            },
+            connected_wifi: {
+              value: 0,
+              timestamp: TIMESTAMP,
+            },
+            os_build: {
+              value: 'PQ2A.190406.000',
+              timestamp: TIMESTAMP,
+            },
+          },
+          cell_plan: {
+            data_plan_10gb: {
+              value: 'true',
+              timestamp: TIMESTAMP,
+            },
+          },
+        },
+      },
+    ];
+
+    await table.insert(rowsToInsert).catch(console.error);
+  });
+
+  after(async () => {
+    await table.delete().catch(console.error);
+  });
+
+  it('should filter with row sample', async () => {
+    const stdout = execSync(
+      `node filterSnippets ${INSTANCE_ID} ${TABLE_ID} filterRowSample`
+    );
+    console.log('stdout', stdout);
+    const result = `Reading data for`;
+    assert.equal(stdout.includes(result), true);
+  });
+
+  it('should filter with row regex', async () => {
+    const stdout = execSync(
+      `node filterSnippets ${INSTANCE_ID} ${TABLE_ID} filterRowRegex`
+    );
+    const result = `Reading data for phone#4c410523#20190501:
+Column Family stats_summary
```

Comment:
This is 🍌🍌🍌.  @bcoe don't you have a fancy way of testing output you wanted us to start using for situations like this?

Reply:
Yeah, this is pretty rough; If we know the data is not going to change between tests, I suggest that we'd use:

https://www.npmjs.com/package/snap-shot-it

It will capture the output on the first run of tests, and then fail tests if the output happens to change in the future (at which point you can eyeball using a diff what has actually changed).

Alternatively, since this is just a smoke test, I would just test with a much smaller regex, e.g.,

```js
/Reading Data for phone#[0-9]c[0-9]{6}#[0-9]{9}/
```

I would definitely avoid putting this large of a fixture inline.

Reply:
For what it's worth, I think snapshots are a great pattern, and have been itching to try them out if the need arises in our client libraries:


https://jestjs.io/docs/en/snapshot-testing

Reply:
Added the snapshots